client/budgedtify/src/app/account/accountmain/leftside/leftside.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { AccountFrontService } from '../../account-front.service';
+import { DialogComponent } from '../../dialog/dialog.component';
+
+@Component({
+  selector: 'app-leftside',
+  templateUrl: './leftside.component.html',
+  styleUrls: ['./leftside.component.scss'],
+})
+export class LeftsideComponent implements OnInit {
+  constructor(
+    private accountService: AccountFrontService,
+    public dialog: MatDialog
+  ) {}
+
+  responsedata: any;
```

Comment:
`any`

Reply:
Please change request to:  

```
  getAllAccounts(): Observable<AllAccounts[]> {
    return this.http.get<AllAccounts[]>('http://localhost:3000/accounts');
  }
```

Then response data type will be `AllAccounts[]`


By the way,

```
export interface AllAccounts {
  title:string;
  description:string;
  currency:string
} 
```

This interface represents single entity, so interface name is misleading. You have change name to `Account`